qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -476,34 +499,70 @@ def export_to_file(
         file_type=FileType.CSV.value,
         coordinates=Coordinate.CYLINDRICAL.value,
     ):
-        """Prepares and saves the driven control in a file.
+        """
+        Prepares and saves the driven control in a file.
 
         Parameters
         ----------
-        filename : str, optional
+        filename : str
             Name and path of the file to save the control into.
-            Defaults to None
         file_format : str
-            Specified file format for saving the control. Defaults to
-            'Q-CTRL expanded'; Currently it does not support any other format.
-            For detail of the `Q-CTRL Expanded Format` consult
-            `Q-CTRL Control Data Format
-            <https://docs.q-ctrl.com/wiki/output-data-formats#q-ctrl-hardware>` _.
+            Specified file format for saving the control. Defaults to 'Q-CTRL expanded'. Currently
+            does not support any other format. For detail of the Q-CTRL expanded format, see below.
```

Comment:
```suggestion
            does not support any other format. For details of the Q-CTRL expanded format, see Notes.
```

Just being a bit more specific because MSG doesn't recommend "below" for location in a text (https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below). Feel free to rephrase it in any way you prefer